server/task/pom.xml

```diff
@@ -70,6 +85,58 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>${scala.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.scala-lang.modules</groupId>
+            <artifactId>scala-java8-compat_${scala.base}</artifactId>
+            <version>0.9.0</version>
+        </dependency>
     </dependencies>
 
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>net.alchim31.maven</groupId>
+                    <artifactId>scala-maven-plugin</artifactId>
+                    <version>3.4.4</version>
```

Comment:
Don't we already have that version?

Reply:
Some projects use older version of scala/scala-maven-plugin.
It should be a task in the update sprint IMHO.

Reply:
did you even tried to use the same version everywhere? It's maybe just a matter of minutes